### ISSUE TITLE
Image resolution for outdoor feature matching

### ISSUE THREAD

**Issue by UditSinghParihar** [Tangshitao/QuadTreeAttention#9]
Hello @Tangshitao ,
    Thanks for providing excellent work on QuadTreeAttention.

I want to point out the 2 bugs in [FeatureMatching/notebooks/demo_single_pair.ipynb](https://github.com/Tangshitao/QuadTreeAttention/blob/master/FeatureMatching/notebooks/demo_single_pair.ipynb) .

1. The first bug is while loading the default config of `KeyError of block_type`, which I have *corrected* by appending configs from [training setup](https://github.com/Tangshitao/QuadTreeAttention/blob/master/FeatureMatching/configs/loftr/indoor/loftr_ds_quadtree.py#L11) into the [cvpr_ds_config.py](https://github.com/Tangshitao/QuadTreeAttention/blob/master/FeatureMatching/src/loftr/utils/cvpr_ds_config.py). I have added following lines in `cvpr_ds_config.py`:  
```
# 5. Quadtree
_CN.COARSE.BLOCK_TYPE = 'quadtree'  
_CN.COARSE.ATTN_TYPE = 'B'  
_CN.COARSE.TOPKS=[32, 16, 16]  
_CN.FINE.BLOCK_TYPE = 'loftr'  
```

3. The second bug I am facing while inferring outdoor weights during resizing operation at the following line:  
`img0_raw = cv2.resize(img0_raw, (img0_raw.shape[1]//8*8, img0_raw.shape[0]//8*8))  # input size should be divisible by 8`  
This creates an issue down the line in the quadtree attention block while reshaping due to size mismatch. I have *resolved* the issue by resizing the image to `(640, 480)` and it is working fine:  
`img0_raw = cv2.resize(img0_raw, (480, 640))`  
But I believe during the training on Megadeth images, you have kept the higher side to be `832` while being divisible by `8`. But during inference, that same process is not working as it working in the original [LoFTR notebook](https://github.com/zju3dv/LoFTR/blob/2122156015b61fbb650e28b58a958e4d632b1058/notebooks/demo_single_pair.ipynb).

**TLDR**: Could you remove the bugs in `demo_single_pair.ipynb` in the **default config** and the **resizing operation in outdoor weights**?

**Comment by Tangshitao** [comment]
Thanks for reminding. I did not modified any codes in demo_single_pair.ipynb from the original repo LoFTR. For the second modification, you should make the image size be a divisible of 32.

**Comment by Tangshitao** [comment]
Do you mean for the training or the testing? If you resize the large side to 832, it is already divisible by 32.
